src/main/java/com/faforever/client/fx/PlatformService.java

```diff
@@ -35,6 +40,10 @@ public PlatformService(HostServices hostServices) {
    */
   
   public void showDocument(String url) {
+    if (url.startsWith(IRC_JOIN_PROTOCOL)) {
+      eventBus.post(new JoinChannelEvent(url.substring(IRC_JOIN_PROTOCOL.length() + 3)));
```

Comment:
hmm, that 3 is a bit weird, maybe we could work with CHANNEL_URL_FORMAT by making that public

Reply:
Yes thought about that as well gonna do it then. 3 is for "://"

Reply:
Is there an advantage in calling `java.openUrl()` from the web view and then dissecting the URL here in the PlatformService back into a channel name, over providing a dedicated method `joinChannel`?